igvc_platform/launch/joystick_driver.launch

```diff
@@ -9,7 +9,7 @@
 <launch>
     <!-- Joystick node -->
     <node name="joy_node" pkg="joy" type="joy_node" output="screen" respawn="false">
-        <param name="dev" type="str" value="/dev/input/js0"/>
+        <param name="dev" type="str" value="/dev/igvc_joystick"/>
```

Comment:
Why are the joystick changes also in here?

Reply:
To prevent configuration conflicts with udev rules and to keep all our peripherals organized

Reply:
But this is a PR for the imu. I don't think it belongs in here.

Reply:
Sure, i'm going to make a separate PR

Reply:
Actually, `devel` contains the most recent udev rules. I just hadn't merged.